src/fastfield/delete.rs

```diff
@@ -45,8 +47,23 @@ impl DeleteBitSet {
             .iter()
             .map(|b| b.count_ones() as usize)
             .sum();
+        let mut tinysets = Vec::new();
+        let mut value_accumulator = 0u64;
+        let byte_count = data.as_slice().len();
+        for index in 0..byte_count {
+            value_accumulator <<= 1;
+            value_accumulator += data.as_slice()[byte_count - index - 1] as u64;
```

Comment:
Using the ByteOrder crate would probably be more readable and more efficient as described below....